### DISCUSSION TITLE
Welcome to mosaic Discussions!

### DISCUSSION THREAD

**Discussion by yeegor** [tilework/mosaic#2] [General]
## Welcome to our talk space! 👋

If you have some questions regarding any part of Mosaic, this is the very place to ask!
If you believe something is not working as expected for you - it might be better to open an issue though.
We hope that your experience with Mosaic is great and we are open for any communication towards it.
Of course, we are glad to discuss the concepts themselves and potential use cases as well!

Let's discuss!

**Comment by sliceem88** [comment]
@alfredsgenkins does this extension mechanism supports hooks and functional components ?
